README.md

```diff
@@ -3,6 +3,7 @@
 
 # Math Magicians: setup
```

Comment:
# Optional
- Please Update the title to `Math magicians: full website` because it is the title of your project

Reply:
## Thanks @OLIPLICHE for reviewing my project!
Really my project's general layout was not matching the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).
I have tried my best to match it as required in the project.

***Cheers and happy reviewing!***